### ISSUE TITLE
if boot_mac doesn't exist, action to create it.

### ISSUE THREAD

**Issue by dnbert** [edolnx/nephology-server-perl#14]
This is an optional thing that I would like to have.

If a machine doesn't exist in our database, we should have the automatic option of forking it to a a new controller and populating the DB with its credentials (or ghost credentials)

**Comment by edolnx** [comment]
That was kind of the idea, when an unknown boot mac is detected it should be registered with the DB and not have a caste assigned to it.  It's then up to a human to assign a caste to the system via the webui

**Comment by dnbert** [comment]
Absolutely, I just didn't see any logic inside of the Boot controller that would handle this, so I bugged it.
